MANIFEST.in

```diff
@@ -1,4 +1,5 @@
 recursive-include demos *.py *.yaml *.html *.css *.js *.xml *.sql README
+recursive-include docs *
```

Comment:
I think this also needs a `prune docs/build` so it doesn't include build output if the docs have been built in this workspace before.